### ISSUE TITLE
[Tensorboard] my training loss is stucked at step 1, but evaluation loss is updating

### ISSUE THREAD

**Issue by dain5832** [tensorflow/models#8561]
- What is the top-level directory of the model you are using: tensorflow1/models/research/object_detection

- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10

- TensorFlow installed from (source or binary): pip

- TensorFlow version (use command below): 1.14.0

- Bazel version (if compiling from source): NA

- CUDA/cuDNN version: NA

- GPU model and memory: NA

- Exact command to reproduce: python object_detection/model_main.py --pipeline_config_path=object_detection/datasets/experiments/training/faster_rcnn_datasets.config --model_dir=object_detection/datasets/experiments/training --num_train_steps=50000 --sample_1_of_n_eval_examples=1 --alsologtostderr

tensorboard --logdir ankylosing/experiments/training --host localhost --port 8088

I'm using model_main.py to train my model. It didn't give me an error and works quite well. However, when I checked tensorboard, my loss_1 and loss_2 graph seemed like not updating after step1.

![image](https://user-images.githubusercontent.com/62231006/82625864-39745300-9c21-11ea-9a44-80e204ade30d.png)

Can you tell me how to fix it?

![image](https://user-images.githubusercontent.com/62231006/82626540-0763f080-9c23-11ea-87a3-a7dd5a8da9cb.png)

My other loss graphs look like this. 
And below is the result I got from each step.

INFO:tensorflow:Calling model_fn.                                                          
I0522 11:32:46.233400  1256 estimator.py:1145] Calling model_fn.                           INFO:tensorflow:Scale of 0 disables regularizer.                                           
I0522 11:32:46.270332  1256 regularizers.py:98] Scale of 0 disables regularizer.    
INFO:tensorflow:Done calling model_fn.                                                     
I0522 11:09:04.982665  1256 estimator.py:1147] Done calling model_fn.                      INFO:tensorflow:Starting evaluation at 2020-05-22T11:09:04Z                                
I0522 11:09:04.999619  1256 evaluation.py:255] Starting evaluation at 2020-05-22T11:09:04Z INFO:tensorflow:Graph was finalized.                                                       
I0522 11:09:05.884253  1256 monitored_session.py:240] Graph was finalized.                 INFO:tensorflow:Restoring parameters from object_detection/ankylosing/experiments/training\model.ckpt-93                                                                              I0522 11:09:05.890208  1256 saver.py:1280] Restoring parameters from object_detection/ankylosing/experiments/training\model.ckpt-93                                                   INFO:tensorflow:Running local_init_op.     
I0522 11:09:07.192721  1256 session_manager.py:500] Running local_init_op.                 INFO:tensorflow:Done running local_init_op.                                                
I0522 11:09:07.348306  1256 session_manager.py:502] Done running local_init_op.            INFO:tensorflow:Performing evaluation on 221 images.                                       
I0522 11:32:15.174471 43460 coco_evaluation.py:236] Performing evaluation on 221 images.   creating index...                                                                          
index created!                                                                             
INFO:tensorflow:Loading and preparing annotation results...                                
I0522 11:32:15.213368 43460 coco_tools.py:115] Loading and preparing annotation results... INFO:tensorflow:DONE (t=0.14s)                                                             
I0522 11:32:15.356982 43460 coco_tools.py:137] DONE (t=0.14s)                              
creating index...                                                                          
index created!                                                                             
Running per image evaluation...                                                            
Evaluate annotation type *bbox*                                                            
DONE (t=7.06s).                                                                            
Accumulating evaluation results...                                                         
DONE (t=0.39s).                                                                             
Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.238             
Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.530             
Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.147             
Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = -1.000            
Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = -1.000            
Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.238             
Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.293             
Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.495             
Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.525             
Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = -1.000            
Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = -1.000            
Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.525            INFO:tensorflow:Finished evaluation at 2020-05-22-11:32:23                                 
I0522 11:32:23.277833  1256 evaluation.py:275] Finished evaluation at 2020-05-22-11:32:23  INFO:tensorflow:Saving dict for global step 93: DetectionBoxes_Precision/mAP = 0.23755923, DetectionBoxes_Precision/mAP (large) = 0.23755945, DetectionBoxes_Precision/mAP (medium) = -1.0, DetectionBoxes_Precision/mAP (small) = -1.0, DetectionBoxes_Precision/mAP@.50IOU = 0.5303363, DetectionBoxes_Precision/mAP@.75IOU = 0.14674938, DetectionBoxes_Recall/AR@1 = 0.29293266, DetectionBoxes_Recall/AR@10 = 0.4948963, DetectionBoxes_Recall/AR@100 = 0.5246189, DetectionBoxes_Recall/AR@100 (large) = 0.5246189, DetectionBoxes_Recall/AR@100 (medium) = -1.0, DetectionBoxes_Recall/AR@100 (small) = -1.0, Loss/BoxClassifierLoss/classification_loss = 0.24961615, Loss/BoxClassifierLoss/localization_loss = 0.4095158, Loss/RPNLoss/localization_loss = 0.15028447, Loss/RPNLoss/objectness_loss = 0.23827589, Loss/total_loss = 1.0476927, global_step = 93, learning_rate = 0.0003, loss = 1.0476927                           I0522 11:32:23.280791  1256 estimator.py:2039] Saving dict for global step 93: DetectionBoxes_Precision/mAP = 0.23755923, DetectionBoxes_Precision/mAP (large) = 0.23755945, DetectionBoxes_Precision/mAP (medium) = -1.0, DetectionBoxes_Precision/mAP (small) = -1.0, DetectionBoxes_Precision/mAP@.50IOU = 0.5303363, DetectionBoxes_Precision/mAP@.75IOU = 0.14674938, DetectionBoxes_Recall/AR@1 = 0.29293266, DetectionBoxes_Recall/AR@10 = 0.4948963, DetectionBoxes_Recall/AR@100 = 0.5246189, DetectionBoxes_Recall/AR@100 (large) = 0.5246189, DetectionBoxes_Recall/AR@100 (medium) = -1.0, DetectionBoxes_Recall/AR@100 (small) = -1.0, Loss/BoxClassifierLoss/classification_loss = 0.24961615, Loss/BoxClassifierLoss/localization_loss = 0.4095158, Loss/RPNLoss/localization_loss = 0.15028447, Loss/RPNLoss/objectness_loss = 0.23827589, Loss/total_loss = 1.0476927, global_step = 93, learning_rate = 0.0003, loss = 1.0476927                                                                                       INFO:tensorflow:Saving 'checkpoint_path' summary for global step 93: object_detection/ankylosing/experiments/training\model.ckpt-93                                                   I0522 11:32:23.324671  1256 estimator.py:2099] Saving 'checkpoint_path' summary for global step 93: object_detection/ankylosing/experiments/training\model.ckpt-93                    INFO:tensorflow:Saving checkpoints for 94 into object_detection/ankylosing/experiments/training\model.ckpt.                                                                           I0522 11:32:39.134387  1256 basic_session_run_hooks.py:606] Saving checkpoints for 94 into object_detection/ankylosing/experiments/training\model.ckpt.

**Comment by dain5832** [comment]
solved. It was because the training loss was set to be printed every 100 steps. Never mind!
